src/handlers/types_planning_updates.rs

```diff
@@ -0,0 +1,208 @@
+            Extra reminders will be sent later this week.");
+        let zulip_req = crate::zulip::MessageApiRequest {
+            recipient: crate::zulip::Recipient::Stream {
+                id: 326132,
+                topic: &format!("{meeting_date_string} planning meeting"),
+            },
+            content: &message,
+        };
+        zulip_req.send(&ctx.github.raw()).await?;
+
+        // Then, we want to schedule the next Thursday after this
+        let mut thursday = today;
+        while thursday.weekday().num_days_from_monday() != 3 {
+            thursday = thursday.succ();
+        }
+        let thursday_at_noon =
+            Utc.from_utc_datetime(&thursday.and_time(NaiveTime::from_hms(12, 0, 0)));
+        let metadata = serde_json::value::to_value(PlanningMeetingUpdatesPingMetadata {
+            date_string: meeting_date_string,
+        })
+        .unwrap();
+        schedule_job(
+            &*ctx.db.get().await,
+            TypesPlanningMeetingUpdatesPing.name(),
+            metadata,
+            thursday_at_noon,
+        )
+        .await?;
+
+        Ok(())
+    }
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct PlanningMeetingUpdatesPingMetadata {
+    pub date_string: String,
+}
+
+pub struct TypesPlanningMeetingUpdatesPing;
+
+#[async_trait]
+impl Job for TypesPlanningMeetingUpdatesPing {
+    fn name(&self) -> &'static str {
+        "types_planning_meeting_updates_ping"
+    }
+
+    async fn run(&self, ctx: &super::Context, metadata: &serde_json::Value) -> anyhow::Result<()> {
+        let metadata = serde_json::from_value(metadata.clone())?;
+        // On the thursday before the first monday, we want to ping for updates
+        request_updates(ctx, metadata).await?;
+        Ok(())
+    }
+}
+
+pub async fn request_updates(
+    ctx: &super::Context,
+    metadata: PlanningMeetingUpdatesPingMetadata,
+) -> anyhow::Result<()> {
+    let gh = &ctx.github;
+    let types_repo = gh.repository(TYPES_REPO).await?;
+
+    let tracking_issues_query = github::Query {
+        filters: vec![("state", "open"), ("is", "issue")],
+        include_labels: vec!["roadmap-tracking-issue"],
+        exclude_labels: vec![],
+    };
+    let issues = types_repo
+        .get_issues(&gh, &tracking_issues_query)
+        .await
+        .with_context(|| "Unable to get issues.")?;
+
+    let mut issues_needs_updates = vec![];
+    for issue in issues {
+        // Github doesn't have a nice way to get the *last* comment; we would have to paginate all comments to get it.
+        // For now, just bail out if there are more than 100 comments (if this ever becomes a problem, we will have to fix).
+        let comments = issue.get_first100_comments(gh).await?;
+        if comments.len() >= 100 {
+            anyhow::bail!(
+                "Encountered types tracking issue with 100 or more comments; needs implementation."
```

Comment:
I would recommend bailing *for that issue* -- e.g., leaving it in the list always. Debugging failures in triagebot requires infra/triagebot team involvement since this error message will be somewhere in our logs (and/or in the db in this case IIRC), which is going to be painful and means you get *zero* pings out rather than pinging excessively for small number of issues.

You could also consider looking at the last updated date on the issue rather than specifically the update date of the last comment, which doesn't run into this problem.

Reply:
>  e.g., leaving it in the list always

This is a good idea.

> consider looking at the last updated date on the issue

This is a better idea and I'll switch to this.